sdk/storage/azure-storage-blob/tests/test_append_blob_async.py

```diff
@@ -328,6 +328,30 @@ async def test_append_block_from_url(self, **kwargs):
             await destination_blob_client.append_block_from_url(source_blob_client.url + '?' + sas,
                                                                 source_length=LARGE_BLOB_SIZE)
 
+    @BlobPreparer()
+    @recorded_by_proxy_async
+    async def test_append_blob_async_copy_source_error_and_status_code(self, **kwargs):
```

Comment:
There should be a sync equivalent test for this test and all the tests you added `test_*_blob_async`